Add docs about provide sulu twig attributes (#577)
sulu/sulu-docs: cookbook/sulu-twig-attributes.rst

Provide Sulu twig template attributes in your controller
========================================================

> In order to use the same ``base`` template for these custom routes, you need to provide the correct

Reviewer: ```suggestion
In order to use the ``base`` template that is used for pages on custom routes, you need to provide the correct
```
